README.md

```diff
@@ -112,25 +112,26 @@ class Todo < ApplicationModel
   after_commit :broadcast_created, on: :create
 
   def broadcast_created
-    ActionCable.server.broadcast("todos:created", id)
+    ActionCable.server.broadcast("todos:created", name)
   end
 end
 ```
 
 2. Configure your Motion component to listen to an ActionCable channel:
 
 ```ruby
-class MyComponent < ViewComponent::Base
+class TopTodosComponent < ViewComponent::Base
   include Motion::Component
 
   stream_from "todos:created", :handle_created
 
-  def initialize
-    @todos = Todo.all.to_a
+  def initialize(count: 5)
+    @count = count
+    @todos = Todo.order(created_at: :desc).limit(count).pluck(:name)
   end
 
-  def handle_created
-    @todos = Todo.all.to_a
+  def handle_created(name)
+    @todos = [name, *@todos.first(@count - 1)]
```

Comment:
Perhaps this demo should be more like

```ruby
def handle_created(id)
  @todos = [Todo.find(id), *@todos.first(@count - 1)]
end
```

Reply:
I used the `name` because it avoids a database call for each component (which I wanted mostly for aesthetic reasons in the README).